src/System.Linq.Expressions/tests/InterpreterTests.cs

```diff
@@ -17,7 +17,8 @@ public static class InterpreterTests
         [Fact]
         public static void VerifyInstructions_Simple()
         {
-            Expression<Func<string, bool>> f = s => s != null && s.Substring(1).Length * 2 > 0;
+            // Using an unchecked multiplication to ensure that a mul instruction is emitted (and not mul.ovf)
+            Expression<Func<string, bool>> f = s => s != null && unchecked(s.Substring(1).Length * 2) > 0;
```

Comment:
The `unchecked` makes sure the expression uses `Mutliply` rather than `MultiplyChecked`. That `Multiply` is compiled to `mul` rather than `mul.ovf` is what the test itself ensures. If there was an optimisation that realised it would be safe to use `shl` (it wouldn't be safe, but f'rinstance) and the test was updated accordingly, the expression tested would still be `Multiply`.